### ISSUE TITLE
[Bug] dbt snowflake no longer works for microbatch

### ISSUE THREAD

**Issue by sarahjryan** [dbt-labs/dbt-snowflake#1248]
### Is this a new bug in dbt-snowflake?

- [X] I believe this is a new bug in dbt-snowflake
- [X] I have searched the existing issues, and I could not find an existing issue for this bug

### Current Behavior

When calling dbt build on incremental models configured with strategy microbatch I'm now (as of afternoon 12th Nov 24) getting this error:

>Compilation Error in model xxx
 >The flag require_batched_execution_for_custom_microbatch_strategy has not be registered.

  > in macro materialization_incremental_snowflake (macros\materializations\incremental.sql)
  > called by model xxx

Checking the dbt docs a change has been made so DBT_EXPERMINENTAL_MICROBATCH no longer affects runtime functionality.  The article refers to the flag specified in the error and says false is default, no need to specify.  Tried setting to true jsut to check & no microbatching occurs.

https://docs.getdbt.com/reference/global-configs/behavior-changes#custom-microbatch-strategy 

### Expected Behavior

The microbatching works w/o error as it did prior to the change, when the env var was used

### Steps To Reproduce

Configure an incremental as microbatch and run for more than one batch (the first batch seems to work w/o issue)

### Environment

```markdown
- OS: windows 11
- Python: 3.10.9
- dbt-snowflake==1.9.0b1
- dbt-core==1.9.0b2
```

**Comment by sarahjryan** [comment]
Actually after restarting vscode (the effect the full remove of env var experimental...) setting flag to true results in no microbatching, setting to false gives the error

**Comment by QMalcolm** [comment]
Hi @sarahjryan, are you stilling running into this issue? If you are, can you run a `pip freeze | grep dbt-adapters`.

To jump a head as to why I'm asking for the dbt-adapters version, we released a bad version of dbt-adapters, `1.11.0`, which caused this issue. We've since _yanked_ that version from PyPI. Moving to dbt-adapters `1.9.0` (which interestingly was released _after_ `1.11.0`) should resolve the issue.

**Comment by QMalcolm** [comment]
@sarahjryan I'm going to close this out as we haven't heard back. If this is still an issue for you, please re-open this issue 🙂
